app/packages/components/src/components/Selection/Option.tsx

```diff
@@ -0,0 +1,126 @@
+import React from "react";
+
+import styled from "styled-components";
+import { useHover } from "@fiftyone/state";
+import { Edit, Check } from "@mui/icons-material";
+import { useTheme } from "@fiftyone/components";
+
+import Tooltip from "@mui/joy/Tooltip";
+
+const Box = styled.div`
```

Comment:
Just thinking out loud based on the discussions we have been having, styled components is used heavily in the App, but I actually prefer CSS modules because they have code hints etc. Styled components are harder to read.

My point is, once we decide how want to style, we should stick with that and slowly remove everything else. Modules, styled components, or something else.

Reply:
agreed on a consensus on styling one way. we had a short discussion on this on slack. since we are moving to shared component lib with mui, both styled components and css modules will get changed / removed. I personally like styled components since you have access to themes, props passing is possible, and I like the inheritance approach to build reusable ones.